tessen

```diff
@@ -2,9 +2,12 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 # Copyright (C) 2021 Ayush Agarwal <ayush at fastmail dot in>
 #
+# vim: set expandtab ts=2 sw=2 sts=2:
+#
 # tessen - a data selection interface for pass on Wayland
 # ------------------------------------------------------------------------------
-# vim: set expandtab ts=2 sw=2 sts=2:
+
+set -euo pipefail
```

Comment:
Please don't use shell "strict" mode. There are lots of caveats to keep in mind while using it and I don't like how it might behave unexpectedly. If you go through the commit history of `tessen`, you'll notice that I disabled shell "strict" mode in 5ca7bc5.